tests/utils/common.go

```diff
@@ -199,20 +200,49 @@ func DisableFeatureGate(c client.Client, featureGate string) (*bool, error) {
 	return &previousValue, nil
 }
 
-func removeString(featureGates []string, featureGate string) []string {
+// SetInsecureRegistry sets CDIConfig InsecureRegistries to mark the registry as allowed to be insecure
+func SetInsecureRegistry(c client.Client, registryURL string) error {
+	parsedURL, err := url.Parse(registryURL)
+	if err != nil {
+		return err
+	}
+	err = UpdateCDIConfig(c, func(config *cdiv1.CDIConfigSpec) {
+		if hasString(config.InsecureRegistries, parsedURL.Host) {
+			return
+		}
+		config.InsecureRegistries = append(config.InsecureRegistries, parsedURL.Host)
+	})
+	return err
+}
+
+// UnsetInsecureRegistry removed the registry from CDIConfig InsecureRegistries so it is not allowed to be insecure
+func UnsetInsecureRegistry(c client.Client, registryURL string) error {
```

Comment:
I think RemoveInsecureRegistry is a better name, an Unset implies you are restoring a previous value which you are not.

Reply:
renamed